pkgs/tools/admin/pulumi-packages/base.nix

```diff
@@ -109,15 +114,19 @@ mkBasePackage ({
 
   cmd = cmdRes;
 
+  subPackages = [
+    "cmd/${cmdRes}"
+  ];
+
   postConfigure = ''
     pushd ..
-
-    chmod +w sdk/
-    ${cmdGen} schema
-
+    chmod +w sdk
+    ${cmdGen} ${cmdGenArgs}
```

Comment:
I find the distinction between `cmdGen` and `cmdGenArgs` a bit artificial. How about merging this into one attribute? It would also enable to have more than one command in the generate build step (e.g. for newer versions of `pulumi-azure-native`, see #260456)